homeassistant/components/sensor/yr.py

```diff
@@ -61,7 +61,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     coordinates = dict(lat=latitude,
                        lon=longitude,
-                       msl=elevation)
+                       msl=int(elevation))
```

Comment:
I am not sure how this can happen ?

 - loading the core config coerces the elevation to a `float`
 - fetching elevation from the api does that too

`hass.config.elevation` should always be a number, so if that's not the case, we should fix it there.